packages/typespec-ts/test/commands/cadl-ranch-list.ts

```diff
@@ -228,8 +228,8 @@ export const modularTsps: TypeSpecRanchConfig[] = [
     inputPath: "azure/core/basic"
   },
   {
-    outputPath: "encode/bytes",
-    inputPath: "encode/bytes"
+    outputPath: "payload/pageable",
```

Comment:
why remove encode bytes test case ?

Reply:
Also, we probably should clear the generated folder in the CI before we run the generate-tsp-only script as the cadl-ranch-list change is not being correctly reflected.

Reply:
> why remove encode bytes test case ?

I will revert this change and it is introduced wrongly because in latest cadl-ranch the cases are changed for encode/bytes.